README.md

```diff
@@ -115,14 +147,29 @@ sbt run
 ```
 
 Navigate to the desired accessibility statement e.g. http://localhost:12346/accessibility-statement/disguised-remuneration
-where disguised-remuneration is the serviceKey defined in the accessibility statement YAML file.
+where disguised-remuneration is the filename of the accessibility statement YAML file with the language and yaml suffix
+removed.
 
-## Running unit and integration tests together
+## Running unit and integration tests
 
 ```
 sbt test it:test
 ```
 
+## Generating a report
+
+It is possible to generate a tab-separated-value (TSV) file containing information on all the accessibility statements
+that exist in the repository. This TSV file can then easily be imported into your favourite spreadsheet application
+for further analysis.
+
+To generate the report, clone the repository locally and run
+
+```
+sbt "generateReport report.tsv"
```

Comment:
Looking at this and the code below, do we need to specify the report name, or should it be `reports.tsv` by default, and can be overwritten?

Reply:
It's configurable to whatever you want. The first arg is taken as the path to the output file.

Reply:
I think Jo's suggestion is to allow `sbt generateReport` which will generate `report.tsv` as a default.

I think that's good so we can avoid the quote marks if we don't actually care about the output filename.